.github/workflows/create-stable-release.yml

```diff
@@ -0,0 +1,116 @@
+# Automatically create new releases from the main branch when there are new commits since the last stable release.
```

Comment:
If a CI creates a new commit using the job's identity, GitHub doesn't start a new pipeline for this commit/PR. Does it start the pipeline after creating the release?